src/test/groovy/ee/tuleva/onboarding/mandate/GenericMandateIntegrationTest.java

```diff
@@ -93,15 +93,12 @@ void testMandateCreation() throws Exception {
 
     var headers = getHeaders();
 
-    var aDto =
-        GenericMandateCreationDto.builder()
-            .details(new WithdrawalCancellationMandateDetails())
-            .build();
+    var aDto = MandateDto.builder().details(new WithdrawalCancellationMandateDetails()).build();
```

Comment:
`aDto` is a very generic name - a data transfer object of _what_ exactly? Prefer to call it `mandate` or `mandateDto` so it would be conceptually understandable

Reply:
`a` prefix is for fixture, but will improve